common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -61,7 +61,10 @@ export class Layouts {
   static readonly dfltText='`1234567890-=\xA7~~qwertyuiop[]\\~~~asdfghjkl;\'~~~~~?zxcvbnm,./~~~~~ '
     +'~!@#$%^&*()_+\xA7~~QWERTYUIOP{}\\~~~ASDFGHJKL:"~~~~~?ZXCVBNM<>?~~~~~ ';
 
-  static readonly DEFAULT_RAW_SPEC = {'F':'Tahoma', 'BK': Layouts.dfltText} as const;
+  // The function baked into keyboards by the current Web compiler creates an
+  // array of single-char strings for BK. Refer to
+  // developer/src/kmc-kmn/src/kmw-compiler/visual-keyboard-compiler.ts.
+  static readonly DEFAULT_RAW_SPEC = {'F':'Tahoma', 'BK': Layouts.dfltText.split('')} as const;
```

Comment:
From an _old_ keyboard (viewable [here](https://github.com/keymanapp/keyman/blob/c8d8e4187aa20f7db7f22c8fa7ccd21a04a2615b/web/src/test/manual/web/us-1.0.js)):

```javascript
BK: new Array("`","1","2","3","4","5","6","7","8","9","0","-","=","","","","q","w","e","r","t","y","u","i","o","p","[","]","\\","","","","a","s","d","f","g","h","j","k","l",";","'","","","","","","\\","z","x","c","v","b","n","m",",",".","/","","","","",""," ","~","!","@","#","$","%","^","&","*","(",")","_","+","","","","Q","W","E","R","T","Y","U","I","O","P","{","}","|","","","","A","S","D","F","G","H","J","K","L",":","\"","","","","","","|","Z","X","C","V","B","N","M","<",">","?","","","","",""," ","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","","",""," ")
```

From the output generated by modern compilers when run live (via `khmer_angkor`):

> (130) ['«', '១', '២', '៣', '៤', '៥', '៦', '៧', '៨', '៩', '០', 'ឥ', 'ឲ', '', '', '', 'ឆ', '', '', 'រ', 'ត', 'យ', '', '', '', 'ផ', '', 'ឪ', 'ឮ', '', '', '', '', 'ស', 'ដ', 'ថ', 'ង', 'ហ', '', 'ក', 'ល', '', '', '', '', '', '', '', '', 'ឋ', 'ខ', 'ច', 'វ', 'ប', 'ន', 'ម', '', '។', '', '', '', '', '', '', '​', '»', '!', 'ៗ', '"', '៛', '%', '', '', '', '(', ')', '', '=', '', '', '', 'ឈ', '', '', 'ឬ', 'ទ', '', '', '', '', 'ភ', '', 'ឧ', 'ឭ', '', '', '', '', '', 'ឌ', …]

It _is_ technically OK in JavaScript to index a string as if it were an array, which is why things have been working.  TS strictness settings don't like it, though.